compose/docker-compose-db.yml

```diff
@@ -1,20 +1,32 @@
-# TODO: figure out if this is needed -- it doesn't seem to make sense in the context of the
-# current version of cerc/ipld-eth-db/Dockerfile, which is only the migrations code, not the DB itself
-# The sibling component docker-compose-db-sharding.yml looks more plausible
 version: "3.2"
+
 services:
+  migrations:
+    restart: on-failure
+    depends_on:
+      ipld-eth-db:
+        condition: service_healthy
+    image: cerc/ipld-eth-db:local
+    environment:
+      DATABASE_USER: "vdbm"
+      DATABASE_NAME: "cerc_testing"
+      DATABASE_PASSWORD: "password"
+      DATABASE_HOSTNAME: "ipld-eth-db"
+      DATABASE_PORT: 5432
+
   ipld-eth-db:
+    image: timescale/timescaledb:latest-pg14
     restart: always
-    image: cerc/ipld-eth-db:local
+    command: ["postgres", "-c", "log_statement=all"]
```

Comment:
This probably isn't exactly what we need (log all statements). I'd just leave out the `command` directive until we have time to configure the db properly.

Reply:
I can do that, this was just what the original file had.